### PR TITLE
Plug: inline namespace, cleanup, build UI earlier

### PR DESCRIPTION
**Pull request by danirabbit**
Review with hide whitespace changes

* No reason for each view to be public static anymore
* Build UI in get_widget prevents blank UI during navigation animation
* Code style
* Box instead of grid
* Pass a ref to permission instead of public static

### CODE REVIEW

File: src/Plug.vala [hunk 1]
```diff
@@ -1,191 +1,143 @@
...
             stack.add_titled (location, LOCATION, _("Location Services"));
 
+            var box = new Gtk.Box (VERTICAL, 0);
+            box.add (infobar);
+            box.add (stack);
+
             var settings_sidebar = new Granite.SettingsSidebar (stack);
 
-            var paned = new Gtk.Paned (Gtk.Orientation.HORIZONTAL);
-            paned.position = 200;
```
Comment (Marukesu) [inline src/Plug.vala:136]: was the removal of position here intentional?

File: src/Plug.vala [hunk 2]
```diff
@@ -1,191 +1,143 @@
...
+            paned = new Gtk.Paned (HORIZONTAL);
             paned.add1 (settings_sidebar);
-            paned.add2 (grid);
+            paned.add2 (box);
+            paned.show_all ();
+
+            stack.notify["visible-child"].connect (() => {
+                infobar.revealed = !permission.allowed && stack.visible_child == firewall;
```
Comment (Marukesu) [inline src/Plug.vala:90]: i would check those in reverse order, there's no need to check if the permission is allowed if we aren't in the firewall page.

File: src/Plug.vala [hunk 3]
```diff
@@ -1,191 +1,143 @@
...
+            });
 
-            main_grid.add (paned);
-            main_grid.show_all ();
+            permission.notify["allowed"].connect (() => {
+                infobar.revealed = !permission.allowed && stack.visible_child == firewall;
```
Comment (Marukesu) [inline src/Plug.vala:94]: same here.

File: src/Plug.vala (hunk 1 above)
Comment (danirabbit) [reply src/Plug.vala:136]: Yeah, settingssidebar already seems to reserve enough space

**Review by Marukesu — APPROVED** [review]

**Pull request merged**